bot.js

```diff
@@ -37,12 +38,40 @@ client.on('ready', () => {
         }
     }, null, true, 'America/New_York');
 
+    const calendarJob = new CronJob('0 0 12 * * *', () => {
+        const newReleasesChannel = client.channels.find(ch => ch.name === 'new-releases');
+        if (newReleasesChannel) {
+            getCalendarData().then( calendarData => {
+                const embed = {
+                    'author': {
+                        'name': 'New Releases',
+                        'url': getCalendarDataSource(),
+                        'icon_url': 'https://yt3.ggpht.com/a-/AN66SAwl4t2Xp-dMiNe7tzRNX5WaVlbwst4emwd4ZA=s900-mo-c-c0xffffffff-rj-k-no',
+                    },
+                    'color': 0xD1002A,
+                    'fields': [],
+                };
+
+                let embedDescription = '';
+
+                Object.keys(calendarData).map( releaseType => {
+                    embed.fields.push({
+                        'name': releaseType,
+                        'value': calendarData[releaseType].map( release => `${release}\n`).join(''),
```

Comment:
What convention do we have for spaces before variables in arrows?

Reply:
I generally have it like `Object.keys(calendarData).map(releaseType => {` or `Object.keys(calendarData).map((releaseType) => {`. Maybe we should setup eslint.